docs/articles_en/learn-openvino/llm_inference_guide/genai-guide-npu.rst

```diff
@@ -9,70 +9,114 @@ This guide will give you extra details on how to utilize NPU with the GenAI flav
 for information on how to start.
 
 Prerequisites
-#############
+#####################
 
 Install required dependencies:
 
 .. code-block:: console
 
    python -m venv npu-env
    npu-env\Scripts\activate
-   pip install optimum-intel nncf==2.11 onnx==1.16.1
+   pip install nncf==2.12 onnx==1.16.1 optimum-intel==1.19.0
    pip install --pre openvino openvino-tokenizers openvino-genai --extra-index-url https://storage.openvinotoolkit.org/simple/wheels/nightly
 
 Export an LLM model via Hugging Face Optimum-Intel
 ##################################################
 
-A chat-tuned TinyLlama model is used in this example. The following conversion & optimization
-settings are recommended when using the NPU:
+As **NPU supports only symmetrically-quantized 4-bit (INT4) models**, make sure to export
+the model with the proper conversion and optimization settings.
 
-.. code-block:: python
+| You may export LLMs via Optimum-Intel, using one of two compression methods:
+| **group quantization** - for both smaller and larger models,
+| **channel-wise quantization** - remarkably effective but for models exceeding 1 billion parameters.
 
-   optimum-cli export openvino -m TinyLlama/TinyLlama-1.1B-Chat-v1.0 --weight-format int4 --sym --group-size 128 --ratio 1.0 TinyLlama
+You select one of the methods by setting the ``--group-size`` parameter to either ``128`` or ``-1``, respectively. See the following examples:
 
-**For models exceeding 1 billion parameters**, it is recommended to use **channel-wise
-quantization** that is remarkably effective. For example, you can try the approach with the
-llama-2-7b-chat-hf model:
+.. tab-set::
+
+   .. tab-item:: Group quantization
+
+      .. code-block:: console
+         :name: group-quant
+
+         optimum-cli export openvino -m TinyLlama/TinyLlama-1.1B-Chat-v1.0 --weight-format int4 --sym --ratio 1.0 --group_size 128
+
+   .. tab-item:: Channel-wise quantization
 
-.. code-block:: python
+      .. code-block:: console
+         :name: channel-wise-quant
+         
+         optimum-cli export openvino -m meta-llama/Llama-2-7b-chat-hf --weight-format int4 --sym --ratio 1.0 --group-size -1
+         
+      .. important::
```

Comment:
It may make sense to put the BKM by Andras on how the CW accuracy can be improved (scale estimation, etc).

Reply:
BKM to improve accuracy for CW model is the following:
1. Update `nncf` to `2.13` version (`pip install nncf 2.13`)
2. Use `--scale_estimation` and accuracy aware quantization: (`--awq` + `--dataset=wikitext2`)

Example:
```
optimum-cli export openvino -m meta-llama/Llama-2-7b-chat-hf --weight-format int4 --sym --group-size -1 --ratio 1.0 --awq --scale-estimation --dataset=wikitext2  Llama-2-7b-chat-hf
```

Reply:
@dmatveev , @TolyaTalamanov I have added info about the BKM in:

https://github.com/openvinotoolkit/openvino/pull/27489/commits/95631aad10b14584d0321879475505ec0e0cc00b

Reply:
`--scale_estimation` requires `--dataset` to be specified, I'd rewrite it like this:

Use `--scale_estimation --dataset=<dataset_name>` and accuracy aware quantization: `--awq`

Reply:
I have changed it: https://github.com/openvinotoolkit/openvino/pull/27489/commits/d2087cf52b438cab970109ea9ccc809a86ee096d